spec/controllers/signatures_controller_spec.rb

```diff
@@ -86,6 +86,17 @@
         end
       end
 
+      it 'doesn\'t send email notification to the petition creator if the petition is already in moderation' do
+        petition_in_moderation = FactoryGirl.create(:petition, state: 'sponsored')
+        sponsor = FactoryGirl.create(:sponsor, petition: petition_in_moderation)
+        signature = sponsor.create_signature(FactoryGirl.attributes_for(:pending_signature, petition: petition_in_moderation))
+        perform_enqueued_jobs do
```

Comment:
Just added it here: https://github.com/alphagov/e-petitions/commit/abc5d4bd5f63e73f9c6f0db5ec2128df5982946a
Thans for the tip!